packages/blocks/src/surface-block/element-model/connector.ts

```diff
@@ -37,6 +51,21 @@ export enum ConnectorMode {
   Curve,
 }
 
+export type ConnectorLabelProps = {
+  lableEditing?: boolean; // local
+  label?: Y.Text;
+  labelDisplay?: boolean;
+  labelXYWH?: XYWH;
+  offsetDistance?: number;
+  color?: string;
```

Comment:
Having a `labelStyle` prop object could help avoiding the conflict with stroke style.